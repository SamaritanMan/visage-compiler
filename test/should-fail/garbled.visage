/**
 * Ensure that garbled statements and declarations are caught and recovered from by the parser.
 * 
 * @test/compile-error
 */

package jfx1;


class A {
    va
    function f() {
    }
}
if (A) {

    var ff;
    *9;
    *9;
    var lll;
}

