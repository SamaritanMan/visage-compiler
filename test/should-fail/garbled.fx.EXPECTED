test/should-fail/garbled.fx:11: There is a garbled class member definition starting here, I have tried to ignore it. Class members can be 'var', 'def', 'function', 'init' or 'postinit' only.
    va
    ^
test/should-fail/garbled.fx:18: There is a garbled expression starting here, I have tried to ignore it.
    *9;
    ^
test/should-fail/garbled.fx:19: There is a garbled expression starting here, I have tried to ignore it.
    *9;
    ^
3 errors
